src/Controller/CocktailController.php

```diff
@@ -54,7 +68,26 @@ public function submitName(Request $request): Response
 
         return $this->render('cocktail/submit.html.twig', [
             'cocktailForm' => $form->createView(),
-            'result' => $cocktailName->getCocktailName()
+            'result' => $this->cocktailNameRequest($cocktailName)
         ]);
     }
+
+    private function cocktailNameRequest(CocktailName $cocktailName): string
+    {
+        $response = $this->httpClient->request('GET', self::URL, [
+            'query' => [
+                's' => $cocktailName->getCocktailName()
+            ]
+        ]);
+
+        $parsedResponse = $response->toArray();
+
+        if(empty($parsedResponse['drinks'])){
```

Comment:
Try in Postman to break API request to check what response is returned on error. Try to pass non existent parameter via URL or something to check what happens. If you get error response, try to handle it here too.